### ISSUE TITLE
Error when trying to play my repo

### ISSUE THREAD

**Issue by slaby93** [ajacksified/song-of-github#37]
![image](https://user-images.githubusercontent.com/12088647/29712787-e0ed8fde-899b-11e7-81ee-501576bf938b.png)

![image](https://user-images.githubusercontent.com/12088647/29712833-1af28b26-899c-11e7-9fce-3e3a5f236a61.png)

**Comment by ssorallen** [comment]
This looks like the same error that was fixed by https://github.com/ajacksified/song-of-github/pull/32.

@ajacksified Did you deploy the fix after #32 was merged?

**Comment by ssorallen** [comment]
This is also a duplicate of #36.
